src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -683,6 +683,75 @@ TEST_F(HMICapabilitiesTest, SetIviCooperating) {
   EXPECT_EQ(true, hmi_capabilities_test->is_ivi_cooperating());
 }
 
+TEST_F(
+    HMICapabilitiesTest,
+    UpdateCapabilitiesDependingOn_ccpuVersion_FromCacheForOld_RequestForNew) {
+  MockApplicationManager mock_app_mngr;
+  event_engine_test::MockEventDispatcher mock_dispatcher;
+  MockApplicationManagerSettings mock_application_manager_settings;
+  std::string ccpu_version = "4.1.3.B_EB355B";
+  std::string ccpu_version_new = "5.1.3.B_EB355B";
```

Comment:
@ydementieiev Use `const` qualifier for all variables which are not changed afterwards.

Reply:
@AByzhynar [done](https://github.com/smartdevicelink/sdl_core/pull/3269/commits/a63d278ff6aea958f03c176be3f160a7dc6b2f24#diff-c1d0a5558de3ab3e31ce8813484ebefeR692)